package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "test": "nodeunit test.js"
+    "test": "./node_modules/.bin/nodeunit test.js"
```

Comment:
You shouldn't need this with `npm run test`. Using `npm run` is supposed to add all node_modules executables to your path automatically.